src/api/projects/queries/getProject/index.js

```diff
@@ -13,14 +13,15 @@ type GetProjectArgs = {
 const resolver = async (
   _: any,
   { projectId }: GetProjectArgs,
-  { request }: Context
+  { request }: Context,
+  ast: any
```

Comment:
You probably forgot to remove that ☝️

Reply:
Yes for this PR, but ast was a test to identify the requested fields in the query to make a dynamic eager loading

Reply:
Let’s talk about this, there’s a way to have different resolvers for each field without having to use the AST :)

Reply:
(We’ll also need args on subfields such as “translation keys” for pagination anyway)